Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -73,7 +73,7 @@
           },
           {
             "name": "Builds",
-            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds",
+            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds/?locator=branch:default:any",
```

Comment:
@omeshp Whats the expected outcome of `LatestBuild`? Or in which cases is it called? As it is now it only returns builds from the default branch. Depending on its use it would also require a locator to also return builds from other branches.